include/deal.II/lac/la_parallel_vector.h

```diff
@@ -1364,6 +1364,13 @@ namespace LinearAlgebra
                         const Vector<Number, MemorySpace> &V,
                         const Vector<Number, MemorySpace> &W);
 
+      /**
+       * Assert that the values stored in the ghost part of the vector are all
+       * zero, which is ensured in the vector operations (add, scale, etc.).
```

Comment:
Perhaps say this instead:
```suggestion
       * zero, which is an invariant of the vector space operations such as the addition of vectors, scaling a vector, etc.
```